tiered-storage/jcloud/src/main/java/org/apache/bookkeeper/mledger/offload/jcloud/OffloadIndexBlock.java

```diff
@@ -20,15 +20,13 @@
 
 import java.io.Closeable;
 import java.io.FilterInputStream;
-import java.io.InputStream;
 import java.io.IOException;
+import java.io.InputStream;
 import org.apache.bookkeeper.client.api.LedgerMetadata;
 import org.apache.bookkeeper.common.annotation.InterfaceStability.Unstable;
 
 /**
- *
```

Comment:
Unrelated change

Reply:
Corrected check style violation. Imports have to be in alphabetical order.